test/e2e/specs/wp-launch-spec.js

```diff
@@ -42,32 +44,37 @@ before( async function () {
 	driver = await driverManager.startBrowser();
 } );
 
-// Tracking issue:
-// https://github.com/Automattic/wp-calypso/issues/50547
-// Potential issue that trigger this failure:
-// https://github.com/Automattic/wp-calypso/issues/50273
-describe.skip( `[${ host }] Launch (${ screenSize }) @signup @parallel`, function () {
+describe( `[${ host }] Launch (${ screenSize }) @signup @parallel`, function () {
 	this.timeout( mochaTimeOut );
 
 	describe( 'Launch a free site', function () {
-		const accountName = dataHelper.getNewBlogName();
 		const siteName = dataHelper.getNewBlogName();
+		console.log( siteName );
```

Comment:
Is there a reason to leave the `console.log` here when tests are running in CI?

Reply:
That was my mistake. Fixing now.

As an aside, I see that we disable the linter rule for `console.log` under `test/e2e`. I believe it should be a warning, to catch instances like this.

Reply:
Removed the line in subsequent revision.